SDWebImage/Core/SDImageIOAnimatedCoder.m

```diff
@@ -29,6 +29,9 @@ @interface SDImageIOCoderFrame : NSObject
 @implementation SDImageIOCoderFrame
 @end
 
+BOOL willTerminated_;
+NSLock *terminatedLock_;
```

Comment:
Don't use global variable. Use instance property instead.

Reply:
In most cases, instances of  'SDImageIOAnimatedCoder'  are not created,  so there seems to be no way to use instance property variables in this place

Reply:
OK